jenkins/prepare-yum-repos

```diff
@@ -27,23 +27,23 @@ STORAGE='http://cortx-storage.colo.seagate.com'
 
 cat <<EOF > /etc/yum.repos.d/motr_last_successful.repo
 [motr-dev]
-baseurl=$STORAGE/releases/cortx/github/main/centos-7.8.2003/last_successful/
+baseurl=$STORAGE/releases/cortx/github/main/centos-7.9.2009/last_successful/
 gpgcheck=0
 name=motr-dev
 enabled=1
 EOF
 
 cat <<EOF > /etc/yum.repos.d/motr_uploads.repo
 [motr-uploads]
-baseurl=$STORAGE/releases/cortx/third-party-deps/centos/centos-7.8.2003-2.0.0-latest/motr_uploads/
+baseurl=$STORAGE/releases/cortx/third-party-deps/centos/centos-7.9.2009-2.0.0-latest/motr_uploads/
```

Comment:
@shailesh-vaidya 

Suggest me  which one to add for lustre and motr-uploads repos in `jenkins/prepare-yum-repos` file.
```
centos-7.9.2009-2.0.0-k8/
centos-7.9.2009-2.0.0-latest/
centos-7.9.2009-custom/ 
```
http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/centos/